We let users enable ccache by themselves. Avoids sandbox violation.
https://bugs.gentoo.org/813768
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,11 +54,6 @@ if(NOT VERSION_NUM)
   endif()
 endif()
 
-find_program(CCACHE_PROGRAM ccache)
-if(CCACHE_PROGRAM)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
-
 if(VERSION_NUM MATCHES untagged)
   project(DevilutionX
     LANGUAGES C CXX)
